utils/i18n.go

```diff
@@ -36,7 +36,10 @@ func InitTranslations() {
 func GetTranslationsBySystemLocale() i18n.TranslateFunc {
 	locale := model.DEFAULT_LOCALE
 	if userLanguage, err := jibber_jabber.DetectLanguage(); err == nil {
-		locale = userLanguage
+		// only set the system locale if is supported, fallback to DEFAULT_LOCALE
```

Comment:
I don't think this is needed?  the next couple of lines should already be doing this.

```
if locales[locale] == "" {
		l4g.Error("Failed to load system translations for '%v' attempting to fall back to '%v'", locale, model.DEFAULT_LOCALE)

		if locales[model.DEFAULT_LOCALE] == "" {
			panic("Failed to load system translations for '" + model.DEFAULT_LOCALE + "'")
		}
	}
```